api/app/controllers/api/v1/auth_controller.rb

```diff
@@ -0,0 +1,57 @@
+module Api
+  module V1
+    class AuthController < BaseController
+      skip_before_action :authorize_request, only: %i[register login]
+
+      def register
+        user = User.new(register_params)
+
+        if user.save
+          user.roles << Role.find_by!(name: "USER")
+          tokens = JwtService.issue_tokens_for(user)
+          render json: {
+            access_token: tokens[:access_token],
+            refresh_token: tokens[:refresh_token],
+            user: {
+              id: user.id,
+              name: user.name,
+              email: user.email,
+              roles: user.roles.select(:id, :name)
+            }
+          }, status: :created
+        else
+          render json: { errors: user.errors.full_messages }, status: :unprocessable_entity
+        end
+      end
+
+      def login
+        user = User.find_by("LOWER(email) = ?", login_params[:email].to_s.downcase)
+
+        if user&.authenticate(login_params[:password])
+          tokens = JwtService.issue_tokens_for(user)
+          render json: {
+            access_token: tokens[:access_token],
+            refresh_token: tokens[:refresh_token]
+          }, status: :ok
+        else
+          render json: { error: "Invalid email or password" }, status: :unauthorized
```

Comment:
before, in some PRs, the team added error messages to the translations, you can do that too, just the English version. It’s not required and maybe won’t even be used in the app, as we most likely won’t need localization for those errors. But in general, people tend to avoid hard-coded string values that aren’t in .yml or some other files, since when the codebase grows, it becomes really painful to update or deal with just raw strings.

As the codebase grows, managing and updating hard-coded strings becomes painful, so it's better to keep them centralized from the start.